tiledb/sm/filter/filter_create.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by EH on 11/21/2021.
```

Comment:
Same as above, use the formal boilerplate

Reply:
Done